mocha.el

```diff
@@ -17,6 +17,7 @@
 ;;; Code:
 (require 'compile)
 (require 'js2-mode)
+(require 'f)
```

Comment:
Nope, I just looked at this one first. Since we are using it in the other two anyways, I am fine with leaving it.